esmvalcore/preprocessor/_time.py

```diff
@@ -456,6 +461,26 @@ def anomalies(cube, period):
         data[indexes] = data[indexes] - ref[cube_coord.points[i]]
 
     cube = cube.copy(data)
+
+    # Standardize the results if requested
+    if standardize:
+        cube_stddev = climate_statistics(cube,
+                                         operator='std_dev',
+                                         period=period)
+        assert cube.ndim == cube_stddev.ndim
```

Comment:
If you think this will ever happen, please raise an exception with a nice error message instead of using `assert`

Reply:
No I do not expect this to happen, so will take it out.

Reply:
Or wait, this was actually a review suggestion by @valeriupredoi before... I'll go for an error message in that case :)